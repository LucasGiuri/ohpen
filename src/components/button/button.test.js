import Button from './button';
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';

describe('Button', () => {
  const text = 'title';
  const dataTestPrefix = 'dataTestPrefix';

  describe('Snapshots', () => {
    it('default snapshot render', () => {
      const tree = renderer.create(<Button text={text} />).toJSON();
      expect(tree).toMatchSnapshot();
    });
  
    it('with custom dataTestPrefix snapshot render', () => {
      const tree = renderer.create(<Button text={text} dataTestPrefix={dataTestPrefix} />).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
