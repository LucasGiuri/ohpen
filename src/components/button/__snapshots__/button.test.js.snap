// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Snapshots default snapshot render 1`] = `
<button
  className="sc-bdfBQB hJYzUd"
  data-test="button-button"
>
  title
</button>
`;

exports[`Button Snapshots with custom dataTestPrefix snapshot render 1`] = `
<button
  className="sc-bdfBQB hJYzUd"
  data-test="dataTestPrefix-button"
>
  title
</button>
`;
